### DISCUSSION TITLE
Integration with CapyMOA?

### DISCUSSION THREAD

**Discussion by cdeterman** [online-ml/river#1753] [Q&A]
I recently reached out to scikit-multiflow with an issue request [here](https://github.com/scikit-multiflow/scikit-multiflow/issues/326).  They stated that development is paused on that module and that I should follow up with [CapyMOA](https://capymoa.org/).  Is there any expectation to integrate with that module moving forward?

**Comment by smastelini** [comment]
Hi @cdeterman, feature importance is certainly something interesting to add in Hoeffding Trees. I am not aware yet of the technical challenges involved, but it is worth a go. I will label this issue as a new feature request and assign myself to it.

**Comment by smastelini** [comment]
As for CapyMOA, that is a different story. River is a standalone Python library for online learning, whereas CapyMOA is a Python wrapper to access MOA (java-based) functionalitites.

PS: skmultiflow is one of the ancestors of River.
